doc/reference/waveform_features.rst

```diff
@@ -0,0 +1,6 @@
+=====================================================================================================
+`waveform_features` - Util functions for computing waveform features like signal-to-noise ratio, etc.
```

Comment:
```suggestion
`waveform_features` - Functions for computing waveform features like signal-to-noise ratio, etc.
```
I rate these not just as utility functions, but proper analysis functions, and thus suggest to remove "util"